views/pages/home.pug

```diff
@@ -8,64 +8,72 @@ block content
     .home__wrapper
       if !userDevice.isMobile
         .home__projects__info
-          .home__projects__info--title Koality
+          .home__projects__info--title
           include ../partials/info-line.pug
           .home__projects__info--number
-        .home__projects__media
 
+        //- TODO: record dedicated video content?
+        .home__projects__media
           .home__projects__media--card1#p1
-            video(controls muted).project__card__video__clip
+            //- video(controls muted).project__card__video__clip
+            img(src="../test-img.png").project__card__video__clip
           .home__projects__media--card2#p2
-            video(controls muted).project__card__video__clip
+            //- video(controls muted).project__card__video__clip
+            img().project__card__video__clip
           .home__projects__media--card3#p3
-            video(controls muted).project__card__video__clip
+            //- video(controls muted).project__card__video__clip
+            img().project__card__video__clip
           .home__projects__media--card4#p4
-            video(controls muted).project__card__video__clip
+            //- video(controls muted).project__card__video__clip
+            img().project__card__video__clip
           .home__projects__media--card5#p5
-            video(controls muted).project__card__video__clip
+            //- video(controls muted).project__card__video__clip
+            img().project__card__video__clip
           .home__projects__media--card6#p6
-            video(controls muted).project__card__video__clip
+            //- video(controls muted).project__card__video__clip
+            img().project__card__video__clip
 
         .home__project__details
           .home__project__overview
             .home__project__overview--heading Overview
             .home__project__overview--description
-            .home__project__overview__links--heading Project Links
+            .home__project__overview__links--heading Links
             .home__project__overview__links--url
               include ../partials/url-icon.pug
-              a.home__project__overview__links--url--link(href="https://example.com" alt="link to live project demo" target="_blank") https://www.example.com
+              a.home__project__overview__links--url--link(href="#" alt="Link to project content" target="_blank")
 
             .home__project__overview__links--github
               include ../partials/github-icon.pug
-              a.home__project__overview__links--url--link(href="https://example.com" alt="link to live project demo" target="_blank") https://www.github.com/project-url
+              a.home__project__overview__links--url--link(href="#" alt="Project GitHub link" target="_blank")
+
+          //- TODO(alex): add this in a dynamic way
```

Comment:
Still need to add this in. Probably make it fade out once a user hovers an object for the first time